### ISSUE TITLE
read_sfpwithout DataFrames?

### ISSUE THREAD

**Issue by brk00** [codles/EEG.jl#24]
Use DataFrames to load a file to a `Electrodes` object sounds like an overkill. Maybe we could write something that don't require DataFrames.jl to work, and someday make DataFrame optional (via Requires.jl). What do yout think?

**Comment by codles** [comment]
Yep that was a hack that I always meant to tidy up. This use of data frames should definitely be removed for something native.
